env/sugarizer.ini

```diff
@@ -22,7 +22,7 @@ consent_need = false
 policy = https://sugarizer.org/policy.html
 
 [client]
-path = ../sugarizer/
```

Comment:
The client app is Sugarizer, not Sugarizer-Server.

Reply:
but when tried ran using this it gives directory not found

when I ran with my changes it ran fine

Reply:
@NikhilM98 see this

![image](https://user-images.githubusercontent.com/100110395/222879608-7298a679-a484-4542-b143-c0d8e7c86373.png)

Reply:
Sugarizer has a directory name `activities`: https://github.com/llaske/sugarizer/tree/master/activities. There must be something wrong with your clone.